packages/flame/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:flutter/gestures.dart';
+
+import '../../extensions.dart';
+
+abstract class _BaseInfo<T> {
+  final T raw;
+
+  _BaseInfo(this.raw);
+}
+
+abstract class _PositionInfo<T> extends _BaseInfo<T> {
+  final Vector2 position;
+  final Vector2 gameWidgetPosition;
+  final Vector2 globalPosition;
+
+  _PositionInfo(
+    this.position,
+    this.gameWidgetPosition,
+    this.globalPosition,
+    T raw,
+  ) : super(raw);
+}
+
+abstract class _LocalPositionOnlyInfo<T> extends _BaseInfo<T> {
```

Comment:
this can be a followup, but my idea was NOT to have this class at all. instead we can use convertGlobalToLocal on behalf of flutter and "fix" so ALL our events have local, global and game coordinates.

Reply:
Fixed